docs/guides/install/add-peer.md

```diff
@@ -2,16 +2,43 @@
 name: Add a peer dependency
 ---
 
-To add an npm package as a peer dependency, directly modify the `peerDependencies` object in your package.json. Running `bun install` will install peer dependencies by default, unless marked optional in `peerDependenciesMeta`.
+
+To add an npm package as a peer dependency, use the `--peer` flag.
+
+```sh
+$ bun add @types/bun --peer
+```
+
+---
+
+This will add the package to `peerDependencies` in `package.json`.
 
 ```json-diff
 {
   "peerDependencies": {
-+   "zod": "^3.0.0"
++   "@types/bun": "^1.0.0"
   }
 }
 ```
 
 ---
 
+Running `bun install` will install peer dependencies by default, unless marked optional in `peerDependenciesMeta`.
+
+```json-diff
+{
+  "peerDependencies": {
+    "@types/bun": "^1.0.0"
```

Comment:
```suggestion
    "@types/bun": "latest"
```

Reply:
hmm maybe we should add some step in the docs to replace latest version

Reply:
this specific example is just showcasing `--peer`, i can use a different package as example to remove confusion though